ak_vendor/report/report_template.html

```diff
@@ -1036,6 +1036,78 @@ <h3 class="analysis__sectionheader">{% trans "Regulatory" %}</h3>
                 {% endif %}
               </div>
 
+              <div class="regulatory__block">
+                {% if analysis.regulatory.mstg %}
+                  <div class="regulatory__category">
+                    <strong class="regulatory__title">{% trans "MSTG" %}</strong>
+                  </div>
+                  <table class="table table--regulatory regulatory__table">
+                    <tbody>
+                      {% for mstg in analysis.regulatory.mstg %}
+                        <tr>
+                          <td>{{ mstg.code }}</td>
+                          <td>{{ mstg.title }}</td>
+                        </tr>
+                      {% endfor %}
+                    </tbody>
+                  </table>
+                {% endif %}
+              </div>
+
+              <div class="regulatory__block">
+                {% if analysis.regulatory.asvs %}
+                  <div class="regulatory__category">
+                    <strong class="regulatory__title">{% trans "ASVS" %}</strong>
+                  </div>
+                  <table class="table table--regulatory regulatory__table">
+                    <tbody>
+                      {% for asvs in analysis.regulatory.asvs %}
+                        <tr>
+                          <td>{{ asvs.code }}</td>
+                          <td>{{ asvs.title }}</td>
+                        </tr>
+                      {% endfor %}
+                    </tbody>
+                  </table>
+                {% endif %}
+              </div>
+
+              <div class="regulatory__block">
+                {% if analysis.regulatory.cwe %}
+                  <div class="regulatory__category">
+                    <strong class="regulatory__title">{% trans "CWE" %}</strong>
+                  </div>
+                  <table class="table table--regulatory regulatory__table">
+                    <tbody>
+                      {% for cwe in analysis.regulatory.cwe %}
+                        <tr>
+                          <td>{{ cwe.code }}</td>
+                          <td>{{ cwe.url }}</td>
```

Comment:
need to have `<a href` defined for urls

Reply:
@subho007 I'm resolving this but please verify at your end as well.